modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -871,7 +878,7 @@ private void updateAssignmentInternal(ConfigurationNotificationEvent<byte[]> ass
                         .thenCompose(v -> storageUpdateHandlerFut)
                         .thenComposeAsync(v -> {
                             try {
-                                return raftMgr.startRaftGroupService(replicaGrpId, newConfiguration);
+                                return raftMgr.startRaftGroupService(replicaGrpId, newConfiguration, raftGroupServiceFactory);
```

Comment:
The same as before, too complicated to hold a factory only to pass it to method start.